src/sagemaker_containers/env.py

```diff
@@ -299,6 +301,12 @@ def log_level(self):  # type: () -> int
         """
         return self._log_level
 
+    @property
+    def framework_module(self):  # type: () -> str
+        """Returns:
+            (str): Name of the framework module to be used for serving."""
```

Comment:
What about: name of the framework module registered in pip or something like that?